README.md

```diff
@@ -1,13 +1,24 @@
-Weever Streaming: Cloud Native, Horizontal Scaling, SFU
-=======================================================
+Weever Streaming: Cloud Native, Horizontal Scaling, WebRTC SFU
+==============================================================
 
 [![Build status](https://github.com/embracer-freemode/weever-streaming/actions/workflows/rust-check.yml/badge.svg)](https://github.com/embracer-freemode/weever-streaming/actions)
 
 Weever Streaming is a Open Source WebRTC SFU (Selective Forwarding Unit).
-It serves for broadcasting, video conferencing, ... etc.
-It's easy to deploy and scalable.
+It serves for broadcasting, video conferencing, or regular video/audio/data streaming ... etc. It's easy to deploy and scale.
 
+You can come up with your own streaming products/platforms easily with it, as long as you are familiar with html, css, javascript, and web service hosting. Client examples can be found at [site/index.html](https://github.com/embracer-freemode/weever-streaming/blob/develop/site/index.html) and [site/management.html](https://github.com/embracer-freemode/weever-streaming/blob/develop/site/management.html).
 
+Currently, Weever Streaming can be deployed with docker-compose in a single machine, or with docker swarm mode to be running on multiple machines. Last but not the least, it can also be deployed to kubernetes shall you wish to go totally cloud native.
+
+Similar or Related Projects We Know of
+========================================
+
+- [Janus](https://janus.conf.meetecho.com/): the general purpose WebRTC server
+- [Jitsi](https://jitsi.org/): Video Conferencing Software
+
+The downside of Janus is that horizontal scaling was a little bit of pain, and Jitsi was designed for video conferencing, it was a bit hard to take it apart and build our own applications with it last time we checked.
```

Comment:
Janus horizontal scaling becomes a bit easier than before after their remote publisher feature is done (otherwise need to manage RTP forwards + Streaming mountpoint). The Janus remote publisher is scheduled after multistream feature. The multistream feature finally landed in Janus v1.0.0 (2022/03/03).

On the other hand, we already got multistream + remote publisher done in Weever Streaming. And Weever Streaming is still much more simpler than Janus when it comes to horizontal scaling.